rollbar-api/src/main/java/com/rollbar/api/payload/data/Person.java

```diff
@@ -15,10 +16,13 @@
 
   private final String email;
 
+  private final Map<String, Object> metadata;
+
   private Person(Builder builder) {
     this.id = builder.id;
     this.username = builder.username;
     this.email = builder.email;
+    this.metadata = builder.metadata != null ? unmodifiableMap(builder.metadata) : null;
```

Comment:
Might not be a big deal, but `unmodifiableMap()` doesn't guarantee that the map is immutable. So this change actually changes Person from an immutable object to a mutable one (though it can only be mutated by mutating the map passed to the builder).

FWIW.